src/HotChocolate/Core/src/Types.Scalars/LatitudeType.cs

```diff
@@ -0,0 +1,167 @@
+        protected override double ParseLiteral(StringValueNode valueSyntax)
+        {
+            if (Latitude.TryDeserialize(valueSyntax.Value, out var value) && value is not null)
+            {
+                return value.Value;
+            }
+
+            throw ThrowHelper.LatitudeType_ParseLiteral_IsInvalid(this);
+        }
+
+        /// <inheritdoc />
+        protected override StringValueNode ParseValue(double runtimeValue)
+        {
+            if (runtimeValue is > Latitude._min and < Latitude._max)
+            {
+                return new StringValueNode(Latitude.TrySerialize(runtimeValue));
+            }
+
+            throw ThrowHelper.LatitudeType_ParseValue_IsInvalid(this);
+        }
+
+        /// <inheritdoc />
+        public override bool TrySerialize(object? runtimeValue, out object? resultValue)
+        {
+            if (runtimeValue is null)
+            {
+                resultValue = null;
+                return true;
+            }
+
+            if (runtimeValue is double d)
+            {
+                resultValue = Latitude.TrySerialize(d);
+                return true;
+            }
+
+            if (runtimeValue is int i)
+            {
+                resultValue = Latitude.TrySerialize(i);
+                return true;
+            }
+
+            resultValue = null;
+            return false;
+        }
+
+        private static class Latitude
+        {
+            public const double _min = -90.0;
+            public const double _max = 90.0;
+            private const int MaxPrecision = 8;
+
+            private const string SexagesimalRegex =
+                "^([0-9]{1,3})°\\s*([0-9]{1,3}(?:\\.(?:[0-9]{1,}))?)['′]\\s*(([0-9]{1,3}" +
+                "(\\.([0-9]{1,}))?)[\"″]\\s*)?([NEOSW]?)$";
+
+            private static readonly Regex _rx =
+                new(SexagesimalRegex, RegexOptions.Compiled | RegexOptions.IgnoreCase);
+
+            public static bool TryDeserialize(string serialized, out double? value)
+            {
+                MatchCollection coords = _rx.Matches(serialized);
+                if (coords.Count > 0)
+                {
+                    var minute = double.TryParse(coords[0].Groups[2].Value, out var min)
+                        ? min / 60
+                        : 0;
+                    var second = double.TryParse(coords[0].Groups[4].Value, out var sec)
+                        ? sec / 3600
+                        : 0;
+                    var degree = double.Parse(coords[0].Groups[1].Value);
+                    var result = degree + minute + second;
+
+                    // Southern and western coordinates must be negative decimals
+                    value = coords[0].Groups[7].Value is "W" or "S" ? -result : result;
+                    return true;
+                }
+
+                value = null;
+                return false;
+            }
+
+            public static string TrySerialize(double serialize)
```

Comment:
The idea of a `Try` method is that you have multiple "return" values. If the serialisation succeeded and the serialised value if it did succeed. You could use a tuple for this but it's more common to use a `bool` as a return value and `out` the other value if it did indeed succeed

```csharp
public static bool TrySerialize(
       double value
       [NotNullWhen(true)] string? serialized) {
       ...
           if(....) {
               serialised ="Foo";
               return false;
           }
           serialised == null;
           return true
       }
```